src/DefaultIcon.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { Dynamic } from './'
+import Dynamic from './layout/Dynamic'
```

Comment:
This turned out to be the cause of an "Element type is invalid" error in FilterList